FBC/ProductsAndServices/ClientsAndAccounts.rdf

```diff
@@ -469,8 +513,20 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>financial service account</rdfs:label>
-		<skos:definition>an account provided by a financial service provider associated with a specific financial product or service</skos:definition>
-		<fibo-fnd-utl-av:adaptedFrom>Barron&apos;s Dictionary of Business and Economics Terms, Fifth Edition, 2012</fibo-fnd-utl-av:adaptedFrom>
+		<skos:definition>account provided by a financial service provider associated with a specific financial product or service</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-pas-caa;GeneralLedger">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-arr;DatedStructuredCollection"/>
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-doc;Record"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;comprises"/>
```

Comment:
(Serious) If a GeneralLedger is a DatedStructuredCollection it should not be a collection of LedgerAccounts which are not DatedCollectionConstituents

Reply:
It could be both, depending on system design.  I've removed the relationship for now, but will do some offline testing to see if that works.